README.md

```diff
@@ -141,7 +141,7 @@ Our [Procfile script](./Procfile) will set up a local example cluster. Start it
 goreman start
 ```
 
-This will bring up 3 etcd members `infra1`, `infra2` and `infra3` and etcd `grpc-proxy`, which runs locally and composes a cluster.
+This will bring up 3 etcd members `infra1`, `infra2` and `infra3` and optionally etcd `grpc-proxy`, which runs locally and composes a cluster.
```

Comment:
Thanks @zcchew1202 Can you please update your commit message to a format that can pass the build e.g. change it to something like `Doc: Add that grpc-proxy is optional in readme` 

https://travis-ci.com/github/etcd-io/etcd/jobs/385962244#L422